### PR TITLE
remove waring, clean linter, list user news

### PR DESCRIPTION
**Pull request by gary-van-woerkens**
- Add JS scripts for Bootstrap
- List team news on index page
- Refactor publish page
- Refactor navigation
- Cleaning

### CODE REVIEW

**Comment by gary-van-woerkens** [comment]
Should wait before merging this one maybe, because getInitialProps in index.js breaks the client side rendering...

**Comment by gary-van-woerkens** [comment]
Can be merged now.

File: pages/index.js [hunk 1]
```diff
@@ -1,46 +1,90 @@
...
+  var dayn = (dt.getDay() + 6) % 7;
+  tdt.setDate(tdt.getDate() - dayn + 3);
+  var firstThursday = tdt.valueOf();
+  tdt.setMonth(0, 1);
+  if (tdt.getDay() !== 4) {
+    tdt.setMonth(0, 1 + ((4 - tdt.getDay() + 7) % 7));
+  }
+  return 1 + Math.ceil((firstThursday - tdt) / 604800000);
+}
+
```
Comment (revolunet) [inline pages/index.js]: you can use [date-fns/getWeek](https://date-fns.org/v2.9.0/docs/getWeek)

File: pages/_document.js [hunk 2]
```diff
@@ -11,11 +11,28 @@ class MyDocument extends Document {
...
+            src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+            crossOrigin="anonymous"
+          ></script>
+
+          <script
+            src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossOrigin="anonymous"
+          ></script>
```
Comment (revolunet) [inline pages/_document.js]: not sure we need all of this... 3rd party JS doesnt play well with React

in last resort, these should be `required` at node level so we dont need to request other servers and leak cookies/tracking info

Comment (gary-van-woerkens) [reply pages/_document.js]: Okay, understood!

File: pages/index.js (hunk 1 above)
Comment (gary-van-woerkens) [reply pages/index.js]: No longer in use in the next version of the project...

**Pull request merged**